documentation-site/examples/tree-view/label-overrides.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import * as React from 'react';
+import {
+  Unstable_StatefulTreeView as StatefulTreeView,
+  TreeLabel,
+  type TreeNodeT,
+} from 'baseui/tree-view';
+import {ChevronRight} from 'baseui/icon';
+import {ChevronDown} from 'baseui/icon';
+
+type CustomNodePropsT = {
+  depth?: number,
+};
+
+const CustomLabel = (node: TreeNodeT) => {
+  const Tag = (depth => {
+    switch (depth) {
+      case 1:
+        return 'h1';
+      case 2:
+        return 'h2';
+      case 3:
+      default:
+        return 'h3';
+    }
+  })(node.depth);
+  // $FlowFixMe
```

Comment:
is it possible to handle the flow error in this documentation example

Reply:
oops fixed!